magento/magento2
# #36346 Add regions for Czech Republic

### Description (*)

Add regions for Czech Republic

### Resolved Issues

1. Resolves magento/magento2#36346

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Czech regions.

### Contribution checklist (*)
